tools/src/main/java/com/powsybl/tools/ToolOptions.java

```diff
@@ -28,9 +28,13 @@
     private final CommandLine line;
     private final FileSystem fileSystem;
 
-    public ToolOptions(CommandLine line, ToolRunningContext context) {
+    public ToolOptions(CommandLine line, FileSystem fileSystem) {
         this.line = Objects.requireNonNull(line);
-        this.fileSystem = Objects.requireNonNull(context).getFileSystem();
+        this.fileSystem = Objects.requireNonNull(fileSystem);
+    }
+
+    public ToolOptions(CommandLine line, ToolRunningContext context) {
```

Comment:
checks if context is null

Reply:
Not necessary here I think: null pointer exception will be thrown anyway by the call to `getFileSystem)`,adding it will only make the following line longer.

Reply:
Okay for me!